docs/gc.md

```diff
@@ -141,15 +141,20 @@ Object ages are tracked for each region with a maximum of 24 possible generation
 
 ![The diagram is explained in the surrounding text](./cr/balanced_regions.png "Region structure and characteristics found in the object heap")
 
- The *eden* space is a set of regions of age 0, which contain the newest objects allocated. Regions of intermediate age serve as survivor regions to receive live objects moved from younger regions. When the eden space is full, a partial garbage collection (PGC) runs to reclaim space, typically by using a *copy forward* operation. In specific cases, a *mark-compact* operation might be used, for example, when there are not enough free survivor regions available. The PGC is a stop-the-world operation, after which new regions are allocated to create a new eden space. A PGC always includes the eden space, but might include older regions. Objects within collectible regions of intermediate age N are evacuated to survivor regions of age N+1. Regions of age 24 are included in PGC collection sets in order to defragment them.
+The *eden* space is a set of regions of age 0, which contain the newest objects allocated. Regions of intermediate age or empty regions serve as survivor regions to receive live objects moved from younger regions. When the *eden* space is full, a partial GC cycle runs to reclaim space, typically by using a *copy forward* operation. In specific cases, a *mark-compact* operation might be used, for example, when there are not enough free survivor regions available. The partial GC cycle is a *stop-the-world* (STW) operation, after which new regions are allocated to create a new *eden* space. A partial cycle always includes the *eden* space, but might include older regions. Objects from collectible regions of age N are moved into another region of the same age N or to an empty region that is assigned an age of N. Then, the ages of all regions across the heap are incremented by 1, except for the maximum age 24 regions. Regions of age 24 are included in partial GC collection sets in order to defragment them.
```

Comment:
'Regions of intermediate age or empty regions serve as survivor regions to receive live objects moved from younger regions'

 can now be even omitted. This one states is very similar and is more correct:
 
Objects from collectible regions of age N are moved into another region of the same age N or to an empty region that is assigned an age of N.